tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/utils/ResourceUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.tools.aerogear.ui.bot.test.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+
```

Comment:
Reading file to string is feature which can be used by many tests from different plugins therefore it's not good to implement it in each plugin which need it. Particular plugin testing some JBT component is not correct place for it.
I think it should be implemented in RedDeer unless it's already is implemented you should search for it. If not in RedDeer then in JBT Integration Tests common plugin that is dependency which can't make any harm to you. guess.